sam/utils/dataframe_functions.py

```diff
@@ -94,7 +94,7 @@ def _index_has_constant_increase(df: pd.DataFrame):
         else:
             return False
 
-    if type(df.index) == pd.core.indexes.datetimes.DatetimeIndex and (
+    if type(df.index) is pd.core.indexes.datetimes.DatetimeIndex and (
```

Comment:
isinstance(df.index, pd.core.indexes.datetimes.DatetimeIndex)

Reply:
Ruben preferred the type() is syntax. I don't mind either way. I think the situations in which they differ almost never occur, and should not in this case, right?